### PR TITLE
Css four column layout

### PR DESCRIPTION
**Pull request by vkahlon**
four-column

### CODE REVIEW

**Comment by vkahlon** [comment]
Screen Shot 2021-12-07 at 9 21 36 PM
 Fixed a typo and updated the screenshot.

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <p class="feature-statement">Accepts only your ideal trips.</p>
+      </div>
+      <div class = "row col-quarter">
+        <img src="images/screen-4.png" alt="screen-4" class="phone-img">
+        <div class="icon-wrapper"><img src="images/bars-icon.png" alt="bars-icon" class="icon-img"></div>
+        <h3 class="features">Stats + Mileage</h3>
+        <p class="feature-statement">Tracks your trips across all apps.</p>
+      </div>
+    </div>
+    </div>
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:39]: I think your indentation got a bit messed up here, there are a couple of divs that seem to have a parent/child relationship, but they are at the same level of indentation 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by vkahlon** [comment]
fixed the indents

File: css-four-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;800&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container over-all-wrapper">
+      <div class="row col-full header">
+        <h3 class="features-how">How it Works</h3>
+      </div>
+      <div class="phone-wrapper">
+        <div class = "row col-quarter">
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:14]: No one element should be both a `row` and a `column` when using layout classes. You likely want this `column-fourth` to have a `row` class element as its child.

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.feature-statement {
+  font-size: small;
+  margin-left: 2rem;
+  padding-top: 9px;
+}
+
+.phone-wrapper {
+  display: flex;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:55]: This looks like you should use the `row` class and just make a class to add the margin that you can use along with the `row` class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by vkahlon** [comment]
Screen Shot 2021-12-09 at 9 00 22 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**